sdk/core/azure-core/src/main/java/com/azure/core/util/tracing/Tracer.java

```diff
@@ -230,12 +229,25 @@
      *
      * @param spanName Name to give the span for the created builder.
      * @param context Additional metadata that is passed through the call stack.
-     *
      * @return The updated {@link Context} object containing the span builder.
      * @throws NullPointerException if {@code context} or {@code spanName} is {@code null}.
      */
     default Context getSharedSpanBuilder(String spanName, Context context) {
         // no-op
         return Context.NONE;
     }
+
+    /**
+     *  Adds an event to the current span with the provided {@code timestamp} and {@code attributes}.
+     *  <p>This API does not provide any normalization if provided timestamps are out of range of the current
+     *  span timeline</p>
+     * <p>Supported attribute values include String, double, boolean, long, String [], double [], long [].</p>
```

Comment:
It would be good to mention what happens if the value doesn't match any of these types. Will it be ignored or will an exception be thrown?